src/pages/sorting/Cards.css

```diff
@@ -0,0 +1,59 @@
+.card {
```

Comment:
if we're gonna use raw CSS, can you please use module.

Just rename this file to `cards.module.css`

then import on the jsx file

```
import style from "./cards.module.css"
```

example like what james did

https://github.com/miracle09110/concept-visualizer/blob/1ad610bc087631e53d388f2f5f9baf2afb1d7d71/src/pages/landingPage/explore/Explore.view.jsx